charts/oes/values.yaml

```diff
@@ -415,17 +415,24 @@ gate:
     #####################################################
     # SAML AUthentication
     #####################################################
-    # saml.jks and metadata.xml are to be mounted as secrets
-    # with names oes-saml-jks and oes-saml-metadata resp.
+    # keytool -genkey -v -keystore oessaml.jks -alias saml -keyalg RSA -keysize 2048 -validity 10000
+    # oessaml.jks and oesmetadata.xml are mounted as secrets, create them as follows
+    # kubectl create secret generic oesmetadataxml --from-file=oesmetadata.xml
+    # kubectl create secret generic oessamljks --from-file=oessaml.jks
+    # kubectl create secret generic samljks-password --from-literal password=changeit 
     saml:
-      enabled: false
+      enabled: true
+      userSource: gate  # Groups will be obtained from SAML
+      keyStore: /opt/spinnaker/saml/oessaml.jks  # The key in this secret must be oessaml.jks
       keyStorePassword: changeit
       keyStoreAliasName: saml
-      issuerId: spind
-      userSource: gate
-
-      jksSecretName: oes-saml-jks
-      metadataSecretName: metadata-xml
+      metadataUrl: /opt/spinnaker/saml/oesmetadata.xml # The key in this secret must be oesmetadata.xml
```

Comment:
I don't think a user need to really worry about the path where metadata file gets mounted on the gate pod. I think we can remove this as a configurable option.